build_tools/cmake/iree_cc_binary.cmake

```diff
@@ -192,4 +192,18 @@ function(iree_cc_binary)
       )
     endif()
   endif()
+
+  # Set up Info.plist properties when building macOS/iOS app bundles.
+  get_target_property(APPLE_BUNDLE ${_NAME} MACOSX_BUNDLE)
+  if (APPLE_BUNDLE)
+    set_target_properties(${_NAME} PROPERTIES
+      MACOSX_BUNDLE_BUNDLE_NAME "${_RULE_NAME}"
+      MACOSX_BUNDLE_GUI_IDENTIFIER "com.google.iree.${_RULE_NAME}"
```

Comment:
Sorry for the late reminder -- do we want to use `org.iree.${_RULE_NAME}` here instead?

Reply:
I'm fine with it. Feel free to send a pull request! :)

Reply:
We actually own the iree.dev domain and should probably use that.